src/parsley.js

```diff
@@ -3,112 +3,97 @@
 //     (c) 2012-2015 Guillaume Potier, Wisembly
 //     Parsley may be freely distributed under the MIT license.
 
-define([
-
-  // ### Requirements
-
-  // Handy third party functions
-  'parsley/utils',
-  // Parsley default configuration
-  'parsley/defaults',
-  // An abstract class shared by `ParsleyField` and `ParsleyForm`
-  'parsley/abstract',
-  // A registry of all Parsley validators (built-in and custom)
-  'parsley/validator_registry',
-  // `ParsleyUI` static class. Handles all UI and UX
-  'parsley/ui',
-  // `ParsleyForm` Class. Handles form validation
-  'parsley/form',
-  // `ParsleyField` Class. Handles field validation
-  'parsley/field',
-  // `Multiple` Class. Extend `ParsleyField` to generate `ParsleyFieldMultiple`
-  'parsley/multiple',
-  // Factory to create Parsley instances (Form, Field or FieldMultiple)
-  'parsley/factory',
-  // Tiny Parsley Pub / Sub mechanism, used for `ParsleyUI` and Listeners
-  'parsley/pubsub',
-  // Default en constraints messages
-  'i18n/en'
-], function (ParsleyUtils, ParsleyDefaults, ParsleyAbstract, ParsleyValidatorRegistry, ParsleyUI, ParsleyForm, ParsleyField, ParsleyMultiple, ParsleyFactory) {
-
-  // Inherit `on`, `off` & `trigger` to Parsley:
-  var Parsley = $.extend(new ParsleyAbstract(), {
-      $element: $(document),
-      actualizeOptions: null,
-      _resetOptions: null,
-      Factory: ParsleyFactory,
-      version: '@@version'
-    });
-
-  // Supplement ParsleyField and Form with ParsleyAbstract
-  // This way, the constructors will have access to those methods
-  $.extend(ParsleyField.prototype, ParsleyAbstract.prototype);
-  $.extend(ParsleyForm.prototype, ParsleyAbstract.prototype);
-  // Inherit actualizeOptions and _resetOptions:
-  $.extend(ParsleyFactory.prototype, ParsleyAbstract.prototype);
-
-  // ### jQuery API
-  // `$('.elem').parsley(options)` or `$('.elem').psly(options)`
-  $.fn.parsley = $.fn.psly = function (options) {
-    if (this.length > 1) {
-      var instances = [];
-
-      this.each(function () {
-        instances.push($(this).parsley(options));
-      });
-
-      return instances;
-    }
-
-    // Return undefined if applied to non existing DOM element
-    if (!$(this).length) {
-      ParsleyUtils.warn('You must bind Parsley on an existing element.');
+import $ from 'jQuery';
```

Comment:
The string needs to be exactly the same as the name of the module that you're requiring in. In this case, it should be `jquery`, as that's the name of the module that jQuery registers itself under on npm, Bower, and so on.

I haven't looked too thoroughly, but I would expect you to also need to add `jquery` as a dep in `package.json`. Atm it's only specified in the Bower package. The reason for this is that the behavior of the module bundler used by the Babel boilerplate basically mimics CJS. In the same way that a Node module will search through `node_modules` for packages, libs built with the boilerplate will do the same.

Lmk if this helps!